package/MDAnalysis/units.py

```diff
@@ -327,6 +328,7 @@
 conversion_factor = {
     'length': lengthUnit_factor,
     'density': densityUnit_factor,
+    'mass': massUnit_factor,
```

Comment:
I think this conversion is useful but you should add a dictionary as given below.

```python
massUnit_factor = {
    'u': 1.0,
    'kg': 1.6605390666e-27
    'g': 1.6605390666e-24,
}
```